diff --git a/scm_irl/__init__.py b/scm_irl/__init__.py
index da3123d..c2e3840 100644
--- a/scm_irl/__init__.py
+++ b/scm_irl/__init__.py
@@ -5,3 +5,12 @@
 __author__ = """Rolando Esquivel Sancho"""
 __email__ = 'rolando.esq@gmail.com'
 __version__ = '0.1.0'
+
+
+from gymnasium.envs.registration import register
+
+register(
+     id="scmirl-v0",
+     entry_point="scm_irl/env:ScmIrlEnv",
+     max_episode_steps=500,
+)
\ No newline at end of file
diff --git a/setup.py b/setup.py
index 16427ba..ff33f7c 100644
--- a/setup.py
+++ b/setup.py
@@ -22,7 +22,8 @@ Links:
 
 from setuptools import setup, find_packages
 
-requirements = ['Click>=6.0', 'numpy', 'gymnasium', 'matplotlib', 'scipy', 'pandas', 'seaborn', 'pygame', 'shapely', 'rasterio', 'opencv-python', 'overpy']
+requirements = ['Click>=6.0', 'numpy', 'gymnasium', 'matplotlib', 'scipy', 'pandas', 'seaborn', 
+                'pygame', 'shapely', 'rasterio', 'opencv-python', 'overpy', "imitation", "wandb"]
 
 setup_requirements = [ ]
 
diff --git a/status/progress.md b/status/progress.md
index 3daea0f..019fc65 100644
--- a/status/progress.md
+++ b/status/progress.md
@@ -158,4 +158,22 @@ Follow established maritime routes:
 Next steps:
 - Use the overpy data (nodes and ways) directly in the observation matrix
 - Handle the ships as a fixed list.
+- Reading material - AIRL: https://arxiv.org/pdf/1710.11248
+
+
+---
+## Week 20 - 14-05-2024:
+
+PPO StableBaselines3 RL execution:
+Reward: `-distance_position -cog_diff -sog_diff`
+<div style="display: flex; justify-content: center;">
+    <video controls src="Screencast 2024-05-14 09:13:11.mp4" title="Title"></video>
+</div>
+
+---
+## Next steps:
+- Add ways to observation matrix.
+- Add scenario handler.
+- Add imitation framework.
+- Add wandb.
 - Reading material - AIRL: https://arxiv.org/pdf/1710.11248
\ No newline at end of file
