diff --git a/scm_irl/__init__.py b/scm_irl/__init__.py
index da3123d..c2e3840 100644
--- a/scm_irl/__init__.py
+++ b/scm_irl/__init__.py
@@ -5,3 +5,12 @@
 __author__ = """Rolando Esquivel Sancho"""
 __email__ = 'rolando.esq@gmail.com'
 __version__ = '0.1.0'
+
+
+from gymnasium.envs.registration import register
+
+register(
+     id="scmirl-v0",
+     entry_point="scm_irl/env:ScmIrlEnv",
+     max_episode_steps=500,
+)
\ No newline at end of file
diff --git a/setup.py b/setup.py
index 16427ba..ff33f7c 100644
--- a/setup.py
+++ b/setup.py
@@ -22,7 +22,8 @@ Links:
 
 from setuptools import setup, find_packages
 
-requirements = ['Click>=6.0', 'numpy', 'gymnasium', 'matplotlib', 'scipy', 'pandas', 'seaborn', 'pygame', 'shapely', 'rasterio', 'opencv-python', 'overpy']
+requirements = ['Click>=6.0', 'numpy', 'gymnasium', 'matplotlib', 'scipy', 'pandas', 'seaborn', 
+                'pygame', 'shapely', 'rasterio', 'opencv-python', 'overpy', "imitation", "wandb"]
 
 setup_requirements = [ ]
 
